core/src/main/java/io/questdb/cutlass/line/AbstractLineSender.java

```diff
@@ -91,33 +74,51 @@ public AbstractLineSender(
     }
 
     public void $(long timestamp) {
-        put(' ').put(timestamp);
-        $();
+        at(timestamp);
     }
 
     public void $() {
+        atNow();
+    }
+
+    @Override
+    public final void atNow() {
```

Comment:
these methods are for the sake of consistency with other clients. I kept the old methods for the sake of backwards compatibility.

Reply:
@jerrinot please don't comment your own changes, this makes it harder to review. If comment is needed, like this one, it should be added in the code.